core/src/main/java/io/jenkins/pluginhealth/scoring/probes/HasUnreleasedProductionChangesProbe.java

```diff
@@ -134,32 +129,26 @@ public ProbeResult doApply(Plugin plugin, ProbeContext context) {
                     List<DiffEntry> entries = df.scan(oldTreeIter, newTreeIter);
 
                     for (DiffEntry entry : entries) {
-                        if (zonedDateTime.isAfter(plugin.getReleaseTimestamp())) {
-                            System.out.println(entry.getNewPath());
-                            filesModifiedAfterRelease.add(entry.getNewPath());
-                        }
+                        tempCommitPath = entry.getNewPath();
                     }
                 }
                 else {
-
-                    System.out.println("in else");
-                    ObjectId treeId = commit.getTree().getId();
+                    RevTree tree = commit.getTree();
                     TreeWalk treeWalk = new TreeWalk(git.getRepository());
```

Comment:
For some reason, I cannot iterate through all the commits.

Either I get the first file in the commit or the last file. 

I went through the documentation, and sample code on GitHub and StackOverflow. But did not find what I am missing here.